lua/neotest/consumers/output_panel/init.lua

```diff
@@ -83,6 +86,16 @@ function neotest.output_panel.toggle()
   end
 end
 
+--- Clears the output panel
+--- >vim
+---   lua require("neotest").output_panel.clear()
+--- <
+function neotest.output_panel.clear()
+  if chan then
+    vim.api.nvim_chan_send(chan, "\x1b[H\x1b[2J")
```

Comment:
This only clears the visible section of the output panel. A `clear` function should completely remove the output.
Can do this with
```lua
  nio.api.nvim_buf_set_option(panel.win:buffer(), "modifiable", true)
  nio.api.nvim_buf_set_lines(panel.win:buffer(), 0, -1, false, {})
  nio.api.nvim_buf_set_option(panel.win:buffer(), "modifiable", false)
```